js/private/js_info.bzl

```
@@ -3,48 +3,71 @@
 JsInfo = provider(
     doc = "Encapsulates information provided by rules in rules_js and derivative rule sets",
     fields = {
-        "declarations": "A list of declaration files produced by the target",
-        "npm_linked_packages": "A list of NpmLinkedPackageInfo providers that are dependencies of this target",
-        "npm_package_stores": "A list of NpmPackageStoreInfo providers from npm dependencies of this target to use as direct dependencies when linking downstream npm_package targets with npm_link_package",
-        "sources": "A list of source files produced by the target",
-        "transitive_declarations": "A list of declaration files produced by the target and the target's transitive deps",
-        "transitive_npm_linked_packages": "A list of NpmLinkedPackageInfo providers that are dependencies of this target and the target's transitive deps",
-        "transitive_npm_package_stores": "A list of NpmPackageStoreInfo providers from npm dependencies of this target and the target's transitive dependencies to use as direct dependencies when linking downstream npm_package targets with npm_link_package",
-        "transitive_sources": "A list of source files produced by the target and the target's transitive deps",
+        "declarations": "A depset of declaration files produced by the target",
+        "npm_linked_package_files": "A depset of files that are that are in npm linked package dependencies of this target",
```

```suggestion
        "npm_linked_package_files": "A depset of files in npm linked package dependencies of this target",
```
(or at least remove the dupe)

---

👍